internal/cmd/resolve/resolve.go

```diff
@@ -48,6 +50,13 @@ Exclude flags could alternatively be set using DEBRICKED_EXCLUSIONS="path1,path2
 Example: 
 $ debricked resolve . `+exampleFlags)
 	cmd.Flags().BoolVar(&verbose, VerboseFlag, true, "set to false to disable extensive resolution error messages")
+	cmd.Flags().IntVar(&regenerate, RegenerateFlag, 0, `Toggles regeneration of lock files between 3 modes:
+0: Default = Only generate new lock files for manifest files without matching lock files.
+1: Regenerate Non-Native: Only generate new lock files for manifest files without matching lock files or matching debricked lock file.
+2: Regenerate: Regenerate all existing lock files and generate new lock files for all manifest files.
+
+Example:
+$ debricked resolve . --regenerate=1`)
```

Comment:
I feel like this documentation is a bit confusing, would love some suggestions for improvement here! :)

Reply:
@filip-debricked One suggestion I have is to check out how we did for the strictness flag in the files find command (https://github.com/debricked/cli/blob/main/internal/cmd/files/find/find.go#L67) I like how the text is structured there to split the options and explanation. 

As for the text, if we focus on just the regeneration part it might be even clearer. So, for example:

Toggles regeneration of already existing lock files between 3 modes:

0 = No regeneration
1 = Regenerates existing non package manager native Debricked lock files
2 = Regenerates all existing lock files

Or something along those lines, maybe?

Reply:
I like it, and changed to your suggestion!